docs/markdown/Dependencies.md

```diff
@@ -393,6 +393,15 @@ foreach h : check_headers
 endforeach
 ```
 
+## dl (libdl)
+
```

Comment:
Should mention:

```
 *(added 0.62.0)*
```

Reply:
Thank you, I'll do a force-push with the fix.